data/builtins/org.thingpedia.builtin.thingengine.builtin/manifest.tt.in

```diff
@@ -153,6 +161,62 @@ class @org.thingpedia.builtin.thingengine.builtin extends @org.thingpedia.volume
   #[minimal_projection=[]]
   #[confirm=false];
 
+  list query configured_device_info(out id : Entity(tt:device)
+                                    #_[canonical={
+                                      base=["name"],
+                                      passive_verb=["named #", "called #"]
+                                    }],
+                                    out description : String
+                                    #_[canonical={
+                                      base=["description"]
+                                    }]
+                                    #[string_values="tt:long_free_text"]
+                                    #[filterable=false],
+                                    out kind : String
+                                    #_[canonical={
+                                      base=["kind"],
+                                      adjective=["#"],
+                                      preposition=["in the # kind"]
+                                    }],
+                                    out version : Number
+                                    #_[canonical={
+                                      base=["version"],
+                                      passive_verb=["version #"]
+                                    }],
+                                    out category : Enum(physical, online, data, system)
+                                    #_[canonical={
+                                      base=["category"],
+                                      adjective=["#"],
+                                      preposition=["in the # category"]
+                                    }],
+                                    out engine_id : String
+                                    #_[canonical={
+                                      base=["engine id"]
+                                    }]
+                                    #[filterable=false],
+                                    out is_transient : Boolean
+                                    #_[canonical={
+                                      base=["is transient"]
+                                    }],
+                                    out auth_type : String
+                                    #_[canonical={
+                                      base=["authentication type"]
+                                    }]
+                                    #[filterable=false])
+  #_[canonical=["information about configured devices",
+                "information about configured entities",
+                "configured devices",
+                "configured entities",
+                "device list",
+                "entity list",
+                "iot devices",
+                "iot entities"]]
+  #_[result={list="here are the cofigured devices"}]
```

Comment:
I would do something like:
```
#_[result={ list="You have ${id}" }]
```
without a `#_[formatted]` annotation.
It will create a single phrase with all the names, which should sound better in voice.